apps/vx-mark/backend/package.json

```diff
@@ -0,0 +1,92 @@
+{
+  "name": "@votingworks/vx-mark-backend",
+  "version": "0.1.0",
+  "private": true,
+  "main": "build/index.js",
+  "types": "build/index.d.ts",
+  "files": [
+    "build"
+  ],
+  "scripts": {
+    "build": "tsc --build tsconfig.build.json",
+    "clean": "rm -rf build tsconfig.tsbuildinfo tsconfig.build.tsbuildinfo",
+    "format": "prettier '**/*.+(css|graphql|json|less|md|mdx|sass|scss|yaml|yml)' --write",
+    "lint": "pnpm type-check && eslint .",
+    "lint:fix": "pnpm type-check && eslint . --fix",
+    "pre-commit": "lint-staged",
+    "start": "node ./build/index.js",
+    "test": "is-ci test:ci test:watch",
+    "test:ci": "jest --coverage --reporters=default --reporters=jest-junit --maxWorkers=7",
+    "test:coverage": "jest --coverage",
+    "test:debug": "node --inspect-brk $(which jest) --runInBand --no-cache",
+    "test:watch": "jest --watch",
+    "type-check": "tsc --build"
+  },
+  "lint-staged": {
+    "*.+(css|graphql|json|less|md|mdx|sass|scss|yaml|yml)": [
+      "prettier --write"
+    ],
+    "*.+(js|jsx|ts|tsx)": [
+      "eslint --quiet --fix"
+    ],
+    "package.json": [
+      "sort-package-json"
+    ]
+  },
+  "dependencies": {
+    "@votingworks/grout": "workspace:*",
+    "@votingworks/logging": "workspace:*",
+    "@votingworks/types": "workspace:*",
+    "debug": "^4.3.1",
+    "dotenv": "^16.0.2",
+    "dotenv-expand": "^9.0.0",
+    "express": "^4.18.0",
+    "fs-extra": "^9.0.1",
+    "js-sha256": "^0.9.0",
+    "multer": "^1.4.2",
+    "uuid": "^8.3.2",
+    "zod": "3.14.4"
```

Comment:
I would imagine many of these deps (e.g. `multer`) and dev deps (e.g. `node-fetch`) aren't necessary  unless I'm missing something. Was this list copied from another app? Maybe start with an empty list and add deps as needed?

Reply:
yeah I copied from vx-scan backend and tried to strip out the stuff that seemed obviously not useful but probably left some unnecessary things in, will see what I can remove.

Reply:
This tool might help: https://www.npmjs.com/package/depcheck